GUIs/ScriptRunnerPyQt5_GUI/ScriptGUIrunner.py

```diff
@@ -11,6 +14,13 @@
 from scripFunc.scripUNIMULTI import *
 from scripFunc.scripWINONLY import *
 
+# The following two `if` statements fix the problem where the application would not scale correctly to the resolution of the monitor
+if hasattr(QtCore.Qt, 'AA_EnableHighDpiScaling'):
```

Comment:
Codacy found an issue: ['QtCore' may be undefined, or defined from star imports: PyUIs.firstconf, scripFunc.scripLINUXONLY, scripFunc.scripUNIMULTI, scripFunc.scripWINONLY, threading (F999)](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5823742)